libs/net/dal/Services/WorkOrderService.cs

```diff
@@ -116,98 +125,13 @@ public IPaged<WorkOrder> Find(WorkOrderFilter filter)
     /// </summary>
     /// <param name="filter">Filter to apply to the query.</param>
     /// <returns>A page of work order items that match the filter.</returns>
-    public IPaged<WorkOrderModel> FindDistinctWorkOrders(WorkOrderFilter filter, JsonSerializerOptions options)
+    public IPaged<WorkOrderModel> FindDistinctWorkOrders(WorkOrderFilter filter)
     {
-        var query =
```

Comment:
This LINQ statement is incorrect.  I don't know why I didn't catch it when Felipe created it.